test/functional/legacy/autocmd_option_spec.lua

```diff
@@ -295,23 +295,23 @@ describe('au OptionSet', function()
 
         nvim.set_option('autochdir', true)
         eq(true, nvim.get_option('autochdir'))
-        expected_combination({'autochdir', '0', '1', 'global'})
+        expected_empty()
       end)
 
       it('should trigger if a number option be set globally', function()
         set_hook('cmdheight')
 
         nvim.set_option('cmdheight', 5)
         eq(5, nvim.get_option('cmdheight'))
-        expected_combination({'cmdheight', 1, 5, 'global'})
+        expected_empty()
```

Comment:
Will move the tests to `api/autocmds_spec.lua` and fix the descriptions if this behaviour is agreed upon.